src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/move/MoveQueryRepository.kt

```diff
@@ -176,9 +176,12 @@ class MoveQueryRepository(@Autowired val jdbcTemplate: JdbcTemplate) {
     }
   }
 
+  /**
+   * Excludes moves without a move_type. This is an indication of bad reporting data in the JSON data feeds.
```

Comment:
Strictly speaking we shouldn't be sent bad data but this the situation we are currently in.  It was a conscious decision to keep accepting these, at least in the short term, as it is an indicator of being supplied bad/poor/incomplete move data.